entrypoint.sh

```diff
@@ -69,9 +92,14 @@ echo "Configuring"
 
 unset RUNNER_TOKEN
 
+# Saving the files in another directory for the possibility to mount them from the host next time
+if [ -d "${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}" ]; then
+  # Quoting (even with double-quotes) the regexp brokes the copying
+  cp -p -r "/actions-runner/_diag" "/actions-runner/svc.sh" /actions-runner/.[^.]* "${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}"
+fi
+
 if [[ ${_DISABLE_AUTOMATIC_DEREGISTRATION} == "false" ]]; then
   trap deregister_runner SIGINT SIGQUIT SIGTERM
 fi
 
-# shellcheck disable=SC2068
-$@ 
+execute_docker_command $@ 
```

Comment:
if the function just runs `$@` then its not needed, just keep `$@` here

Reply:
remove $@ and exit 0 from line 73-74, and just wrap the rest of entrypoint.sh in an `else`, that will let you have a single $@ and naturally exit

Reply:
I was frequently taught about removing the block indentation where it is possible because

1. Indents eat space for the symbols in the line (in case of the limit of the line length)
2. They are needed to be considered always in some specific cases

So there is my try to reduce the indentation. But if you are thinking that it must be removed so be it

Reply:
I think its fine to do an if else and keep it indented. I also agree there but my preference for one $@ and one exit 0 is higher on my list

i think it should make it much easier to read in honesty

Reply:
Done